Directory.Build.props

```diff
@@ -21,6 +21,8 @@
     <ArtifactsBinDir>$(ArtifactsConfigurationDir)bin\</ArtifactsBinDir>
     <PackageOutputPath>$(ArtifactsDir)build\</PackageOutputPath>
 
+    <InstallersOutputPath>$(RepositoryRoot)artifacts/bin/$(Configuration)/installers/</InstallersOutputPath>
```

Comment:
Revert changing where this property is set; it's only used in SharedFramework.wixproj.